README.md

```diff
@@ -278,11 +279,39 @@ Bootstrap JS are loaded.
 
 > **Note**: In general, this is a bad idea.
 
+### Images
+
+Don't forget to remove images that are no longer used.  Try to keep the images
+sorted in the local `images/` directory, with names that naturally group related
+images together in alphabetical order; for instance prefer
+`settings-file-share.png` and `settings-proxies.png` to
+`file-share-settings.png` and `proxies-settings.png`.
```

Comment:
This is a nice convention! The one I've been using with EE docs is to name images after the article they are supposed to be used and append a number, signifying the order in that article.

Something like `install-ubuntu-1.png`, `install-ubuntu-2.png`.
You could argue that this convention doesn't make sense if you're trying to reuse images, but from experience most images are only used in a single article.

Reply:
Hi Joao.  Thanks a lot for your kind words!

Yes, I'll add that conventions too.  In the current case, I still prefer named suffixes, as there's no idea of sequence at all, but I do get your point.